set.mm

```diff
@@ -775592,6 +775639,128 @@ additional condition (deduction form).  See ~ ralbidc for a more
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Unordered and ordered pairs
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( The singleton of the universal class is the empty set.  (Contributed by
+     Zhi Wang, 19-Sep-2024.) $)
+  vsn $p |- { _V } = (/) $=
+    ( cvv wcel wn csn c0 wceq vprc snprc mpbi ) AABCADEFGAHI $.
+
+  ${
+    $d A x z $.  $d A y z $.  $d B x $.
+    $( "At most one" element in a singleton.  (Contributed by Zhi Wang,
+       19-Sep-2024.) $)
+    mosn $p |- ( A = { B } -> E* x x e. A ) $=
+      ( csn wceq cv wcel wmo wtru wrmo rmosn rmotru mpbir eleq2 mobidv mpbiri )
+      BCDZEZAFZBGZAHSQGZAHZUBIAQJIACKAQLMRTUAABQSNOP $.
+
+    $( "At most one" element in an empty set.  (Contributed by Zhi Wang,
+       19-Sep-2024.) $)
+    mo0 $p |- ( A = (/) -> E* x x e. A ) $=
+      ( c0 wceq cvv csn cv wcel wmo vsn eqcomi eqeq1 mpbiri mosn syl ) BCDZBEFZ
+      DZAGBHAIPRCQDQCJKBCQLMABENO $.
+
+    $( Two ways of expressing "at most one" element in a class.  (Contributed
+       by Zhi Wang, 19-Sep-2024.) $)
+    mo0sn $p |- ( E* x x e. A <-> ( A = (/) \/ E. y A = { y } ) ) $=
+      ( vz cv wcel wmo c0 wceq csn wex wo nfv eleq1w cbvmow wn wa wb wal weu
+      neq0 anbi1i df-eu eu6 dfcleq velsn bibi2i albii sylbbr eximi sylbi expcom
+      3bitr2i orrd mo0 mosn exlimiv jaoi impbii bitri ) AECFZAGDEZCFZDGZCHIZCBE
+      ZJZIZBKZLZVAVCADVADMVCAMADCNOVDVJVDVEVIVEPZVDVIVKVDQZVCVBVFIZRZDSZBKZVIVL
+      VCDKZVDQVCDTVPVKVQVDDCUAUBVCDUCVCDBUDUMVOVHBVHVCVBVGFZRZDSVODCVGUEVSVNDVR
+      VMVCDVFUFUGUHUIUJUKULUNVEVDVIDCUOVHVDBDCVFUPUQURUSUT $.
+  $}
+
+  ${
+    $d A f g $.
+    $( Variant of ~ mof0 with stronger requirements on distinct variables.
+       (Contributed by Zhi Wang, 19-Sep-2024.) $)
+    mof0w $p |- E* f f : A --> (/) $=
+      ( vg c0 cv wf wmo wa wceq wi wal f00 simplbi eqtr3 syl2an gen2 feq1 mpbir
+      mo4 ) ADBEZFZBGUAADCEZFZHTUBIZJZCKBKUEBCUATDIZUBDIZUDUCUAUFADIZATLMUCUGUH
+      AUBLMTUBDNOPUAUCBCADTUBQSR $.
+  $}
```

Comment:
Not needed. Unless it requires fewer axioms ?

Reply:
Marked ALT in https://github.com/metamath/set.mm/pull/4220/commits/03dc8ac38062d8146d678dbda62db0b0f52f54de